test/support/BaseSpec.scala

```diff
@@ -38,4 +38,13 @@ class BaseSpec extends PlaySpec with GuiceOneAppPerSuite with Injecting {
 
   def asDocument(rawContent: String): Document = Jsoup.parse(rawContent)
 
+  val legalStatusData =
```

Comment:
Given that this is only used in the one spec `EditRecordViewSpec` I would suggest that it goes there, instead.

Reply:
And I have also renamed this to `legalStatusRefereneData`

Reply:
Confirmed (both).